context-propagation/src/main/java/io/micrometer/context/ThreadLocalAccessor.java

```diff
@@ -32,35 +31,38 @@
 public interface ThreadLocalAccessor<V> {
 
     /**
-     * The key to associate with the ThreadLocal value. This is the key under which the
-     * value is saved within a {@link ContextSnapshot} and the key under which it is
-     * looked up.
+     * The key to associate with the ThreadLocal value when saved within a
+     * {@link ContextSnapshot}.
      */
     Object key();
 
     /**
-     * Return the current {@link ThreadLocal} value, or {@code null} if not set. This
-     * method is called in two scenarios:
+     * Return the current {@link ThreadLocal} value, or {@code null} if not set.
```

Comment:
```suggestion
     * Return the current {@link ThreadLocal} value.
```

Reply:
We don't know whether obtaining null means "not set" or "set to null", but after reading the ThreadLocal we actually know it is initialised. I'd just skip this mention.